python/lsst/pipe/tasks/functors.py

```diff
@@ -1124,6 +1123,14 @@ def instFluxErrToMagnitudeErr(self, instFlux, instFluxErr, localCalib, localCali
 
 class LocalNanojansky(LocalPhotometry):
     """Compute calibrated fluxes using the local calibration value.
+
+    See also
+    --------
+    LocalPhotometry
+    LocalNanojansky
+    LocalNanojanskyErr
+    LocalMagnitude
+    LocalMagnitudeErr
```

Comment:
I think this might be a bit overkill. You can at least remove `LocalPhotometry`, because the docs always have an automatic link to the base class.